charts/openshift-console-plugin/templates/configmap.yaml

```diff
@@ -20,5 +20,18 @@ data:
         ssl_certificate     /var/cert/tls.crt;
         ssl_certificate_key /var/cert/tls.key;
         root                /usr/share/nginx/html;
+
+        # Prevent caching for plugin-manifest.json
+        location = /plugin-manifest.json {
+          add_header Cache-Control 'no-cache, no-store, must-revalidate, proxy-revalidate, max-age=0';
+          add_header Pragma 'no-cache';
+          add_header Expires '0';
+        }
```

Comment:
I'm not against disabling caching for plugin manifest, but I'd also like to understand why this fails in console when the plugin service responds with a 304 status. We should make sure our plugin proxy handles that properly on the console side. Not all plugins will use the template.

Reply:
@jhadvig @spadgett Wasn't this the issue that our backend shows an error for all non-200-OK responses?

That was fixed by @TheRealJon with this PR https://github.com/openshift/console/pull/13272, or?

Should we backport that fix?

Reply:
Maybe related to [this bug](https://issues.redhat.com/browse/OCPBUGS-19875)? Before this was fixed, any response code other than 200 from the plugin service would result in an error response from the console backend. We were not forwarding 304 responses.

Reply:
Yes, definitely backport, otherwise caching will not work properly for plugin assets.